README.md

```diff
@@ -25,7 +25,6 @@ $client = new \OpenEuropa\CdtClient\ApiClient(
     new \GuzzleHttp\Client(),
     new \Http\Factory\Guzzle\RequestFactory(),
     new \Http\Factory\Guzzle\StreamFactory(),
-    new \Http\Factory\Guzzle\UriFactory(),
```

Comment:
This factory turned out to be unnecessary, it was used by `oe_search` to filter through GET params.